content/tracing/languages/cpp.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Tracing C++ Applications
+kind: Documentation
+aliases:
+- /tracing/cpp/
+- /tracing/setup/cpp/
+further_reading:
+- link: "https://github.com/DataDog/dd-opentracing-cpp"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+- link: "tracing/advanced_usage/?tab=c"
+  tag: "Advanced Usage"
+  text: "Advanced Usage"
+---
+
+<div class="alert alert-warning">
+The APM tracer for C++ applications is in Open Public Beta.
```

Comment:
Let's change this disclaimer. Now we are GA with the C++ Tracer so no reason to say anything. What we should add here (I don't know if an alert is good or not) is something like: "C++ does not provide integrations for OOTB instrumentation, but it's used by Proxy tracing such as Envoy and nginx."

The idea is to set the right expectations for users, because in the case of C++ we don't have integrations, made exception of Envoy and nginx.

Reply:
noted